web/src/compose_recipient.ts

```diff
@@ -340,6 +341,19 @@ export function initialize(): void {
         tippy_props: {
             offset: [-10, 5],
         },
+        on_show_callback(instance) {
+            instance.popper.addEventListener("keydown", (event) => {
+                if (event.key !== "Enter") {
+                    return;
+                }
+                if (
+                    document.activeElement?.id === "private_message_recipient" &&
+                    composebox_typeahead.private_message_typeahead.shown
+                ) {
+                    composebox_typeahead.toggle_ignore_typeahead_keyup(true);
+                }
+            });
+        },
```

Comment:
Can you add a block comment for this block explaining exactly what it does and why?